scripts/iam-tools/check_sso_http_status.py

```diff
@@ -20,42 +20,52 @@
 class CheckStatus(object):
     """ Class to check HTTP status code from IDP host. """
 
+
     def __init__(self):
         self.sso_status_code = None
 
 
-    @staticmethod
-    def check_http(ops_idp_host):
-        """ Check HTTP status code returned by the IDP host. """
+    def check_http(self, host):
+        """ Check HTTP status codes returned by the IDP host. """
 
+        url_path = 'https://' + host + '/status.php'
 
-        url_path = 'https://' + ops_idp_host + '/status.php'
-        check_status = requests.get(url_path)
-        sso_status_code = check_status.status_code
+        try:
+            check_status = requests.get(url_path)
+            sso_status_code = check_status.status_code
+            print(sso_status_code, 'status code for', host)
+            return sso_status_code
 
-        return sso_status_code
+        except requests.exceptions.RequestException as request_exception:
+            print(request_exception)
+            return self.sso_status_code
```

Comment:
consider just "return None". self.sso_status_code seems useless and having both a self.sso_status_code and a sso_status_code seems needlessly confusing.